client/src/components/Banner.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {Button} from 'lucid-ui';
+import {Link} from "react-router-dom";
+
+
+
+function signOut(){
```

Comment:
videopage js code for signing out was moved here in banner

Reply:
yep just saw that

Reply:
im not sure what we should do about line 9, perhaps another solution?